conda/hail/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = 'hail' %}
+{% set version = '0.2.61' %}
```

Comment:
How does versioning work here? Shouldn't the package include the commit hash as the version or so, given that we would only increment the semantic version (0.2.61) whenever that happens upstream?

Reply:
I currently set it up to override the existing package in the channel. But good point, it should be better to append the commit hash!